README.md

```diff
@@ -10,6 +10,13 @@ pip install bukowski
 uv pip install bukowski
 ```
 
+Alternatively, use [`uv tool`](https://docs.astral.sh/uv/guides/tools/#installing-tools) to install into a self-contained virtual environment so that bukowski can be invoked without having to create or activate a virtual environment: 
+
+```shell
+uv tool install "git+https://github.com/ninoseki/bukowski.git"
```

Comment:
Thanks but I think ` uv tool install bukowski` is enough. (Don't need to use the git+https URL)

Reply:
Doh, you are completely right, I somehow missed that you have it on pypi.

I've updated the readme snippet.